Unreal/SystemLocalized/det/UnrealI.det

```diff
@@ -68,13 +68,13 @@ MenuName="Gassack"
 
 [GiantGasbag]
 ; EN: NameArticle=" the "
-NameArticle=" der "
+NameArticle=" ein "
```

Comment:
This and following ones may need some discussion on the article use in the original:
Some use the, the Giant Gasbag is clearly a boss (Nali castle), should be perhaps remain "der Riesige Gassack" instead?

Reply:
The problem with that is that there's no "Der Riesige". The name is "Riesiger Gassack", so "ein" would be the only grammatically sound article.

Reply:
@SlyMk3 The problem with that is that there's no "Der Riesige". The name is "Riesiger Gassack", so "ein" would be the only grammatically sound article.